docs/ci-cd-environment/nested-virtualization.md

```diff
@@ -0,0 +1,46 @@
+# Nested virtualization
+
+Linux based virtual machines support nested virtualization.
+
+To check that nested virtualization is supported , we suggest
+issuing `grep -cw vmx /proc/cpuinfo`. The resulting number 
+will be greater than 0 since virtualization flag is present 
+on all VM cpu cores.
+
+Nested virtualization can be managed through `libvirt`, 
+which is already preinstalled.
+
+Predefined default network for nested virtualization is `192.168.123.0/24`.
+The base VM provides `virbr0` interface, having IP address `192.168.123.1`.
```

Comment:
If I understand the context right, I would not go for "..having IP address `192.168.123.1`." but "The base VM provides `virbr0` interface, and the following IP address: `192.168.123.1`."